rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,139 @@
+The Data Browser project details page will add a "To cite this project please copy this link" item.
+This "stable URL" will link back to the production site project page using the project's UUID (e.g. https://data.humancellatlas.org/explore/projects/cc95ff89-2e68-4a08-a234-480eca21ce79).
+The URL refers to the “live" view of the project and is therefore subject to additions and updates (e.g. corrections) of data and metadata. However, these are expected to be infrequent and should not affect existing primary data.
+If an existing project is deleted and re-ingested then the cited project UUID would become invalid. If such re-ingestion is allowed then a means must be provided to redirect the original "stable URL" to the new version of the project.
+Note: Scientists are *already* citing such project based URLs in publications.
+
+See the "Unresolved Questions" section as to whether or not a formal DOI is required for Phase 1.
+
+### Phase 2
+This is designed to satisfy the data contributor requirements for User Story #5.
+This allows the Data Operations team to make a “Data Release” (i.e. a curated data set) where the data release itself and/or the projects contained within are citable.
+A Data Release must be immutable.
+The Data Browser must provide users with access to each Data Release (i.e. immutable version of data) in addition to the “live/latest” view.
+The Data Browser must be able to provide users with a means to download the data and metadata associated with a Data Release.
+Since the Matrix Service does not support the ability to process older versions of input files, the output files from the Matrix Service *must* be stored as part of the immutable data set.
+The Data Store "collections" API provides a suitable means for recording the contents of a Data Release.
+
+### Phase 3
+This is designed to satisfy the data consumer requirements for User Story #5.
+This also fully satisfies User Story #2 by allowing a data consumer to cite an arbitrary set of data which spans multiple projects.
+The data consumer must be able to create discrete versions of their selected data and the Data Browser must provide users access to each version.
+The data consumer must be able to share a citable link to any of the versions they have created.
+
+### Implementation notes for DOI support
+A DOI provides a link of the form https://doi.org/xxxx which resolves via the hosting [Registration Agency](https://www.doi.org/registration_agencies.html) or [open-access repository](https://en.wikipedia.org/wiki/Open-access_repository).
+A DOI is the most commonly accepted means for citing documents and data in scientific publications.
+
+It is acceptable for a DOI to resolve to an external website (e.g. BioStudies, Zenodo, etc.) which could then provide a URL to the project in the Data Browser as well as the ability to store point-in-time copies (i.e. versions) of relevant files such as the download manifest, metadata tsv, matrix output file, etc.
```

Comment:
I think this implies that the DCP must obtain it's own DOI prefix (e.g https://www.crossref.org/get-started/content-registration/) and would exclude solutions such as Zenodo (and Biostudies?). The potential advantage of using an external repository that provides versioning is that we could store manifests there for multiple versions of a project instead of being forced to implement those views within the DCP itself. i.e. The user would always see the latest view of a project in the Data Browser but the (DOI) citation link would take them to an external DOI repository where they could view older versions of the project and where manifests for those older versions would be stored, thus enabling the user to download the older manifest from the external repository and use the HCA CLI to retrieve the specified versions of files. I recall seeing at least one other genomic data site that use Zenodo (can't remember which one). @lauraclarke I need some guidance from PLs or Oversight as to whether the "the DOI resolver takes a user directly to a project page on the HCA-DCP browser" is a hard requirement.

Reply:
@theathorn @morrisonnorman we should work with UX to come up with pros and cons of using an external authority vs setting up the DCP as a DOI assigning entity and then if that effort doesn't give us a clear answer we should ask Oversight for their view

Reply:
Added to Unresolved Questions.

Reply:
@morrisonnorman If your concerns have been addressed, may I resolve this conversation?

Reply:
@theathorn I still have some concerns over the sustainability of using a 3rd party site for storing manifests and minting DOI's. However, I think that is a different concern. Conversation resolved.